### ISSUE TITLE
Mark commit changes vs. file as viewed

### ISSUE THREAD

**Issue by Timppe** [microsoft/vscode-pull-request-github#5796]
When viewing a PR commit by commit (i.e. from Commits directory), it would be very convenient if the "Mark file as viewed" would apply only to the commit at hand. Meaning that, if a file has been modified in 2 separate commits, marking the file as viewed in commit number 1 wouldn't mark the file as viewed in commit number 2, and subsequently not the whole file as such in the PR (as seen from the Files directory). Only after all commits made to a file has been "marked as viewed", the entire file would be marked as viewed.

The functionality would be better called "Mark commit changes as viewed" in the Commits directory, rather than the PR-wide "Mark file as viewed" which could remain as such in the Files directory.

**Comment by alexr00** [comment]
I can see the utility of this, but this is not supported by GitHub.com or by GitHub's API. Marking as out of scope for now.
